.gitignore

```diff
@@ -3,4 +3,4 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
-
+.env
```

Comment:
Yes. It was supposed to ignore some files and folder when pushing to GitHub. It came with the branch I cloned